clients/config_client/config_client_test.go

```diff
@@ -97,6 +97,22 @@ func cretateConfigClientTest() ConfigClient {
 	return client
 }
 
+func createConfigClientWithScheme() ConfigClient {
```

Comment:
This func is not necessary.The default value of Scheme in the current PR is HTTP